language/move-vm/runtime/src/runtime.rs

```diff
@@ -177,24 +264,27 @@ impl VMRuntime {
         module: &ModuleId,
         function_name: &IdentStr,
         ty_args: Vec<TypeTag>,
-        args: Vec<Value>,
+        args: Vec<Vec<u8>>,
```

Comment:
Should we rename this to `execute_function_ext` or something else to distinguish it from the "normal", internal call case?

Reply:
Not sure I understand what "normal" internal call case we're trying to disambiguate from? Could you elaborate?

Reply:
E.g., calling `M::foo()` from `M::bar()`

Reply:
Internal function calls are handled by the interpreter loop. `fn execute_function` is only used when you are calling from an outer environment.